tests/unit/pluginhandler/test_pluginhandler.py

```diff
@@ -824,6 +829,100 @@ def test_prime(self):
         self.assertThat(self.handler.next_step(), Equals(None))
 
 
+class ScriptletSetVersionTestCase(unit.TestCase):
```

Comment:
you could just create `tests/unit/pluginhandler/test_scriptlets.py` and put this in there, not a blocker though

Reply:
Good call, done.